File: beginner/python/fizzbuzz.py
def fizzbuzz(num):
  for _, ind in enumerate(range(1, num + 1)):
    emit = ind
    if ind % 3 == 0 and ind % 5 == 0:
      emit = 'fizzbuzz'
    elif ind % 3 == 0:
      emit = 'fizz'
    elif ind % 5 == 0:
      emit = 'buzz'
    print(emit)

fizzbuzz(60)

File: beginner/javascript/fizzbuzz.js
function fizzbuzz(num) {
  for (let i = 1; i <= num; i++) {
    let emit = i;
    if (i % 3 === 0 && i % 5 === 0)
      emit = 'fizzbuzz'
    else if (i % 3 === 0)
      emit = 'fizz'
    else if (i % 5 === 0)
      emit = 'buzz'
    console.log(emit)
  }  
}

fizzbuzz(60)